panel/pane/markup.py

```diff
@@ -461,7 +460,7 @@ def _transform_object(self, obj: Any) -> dict[str, Any]:
             html = markdown.markdown(
                 obj,
                 extensions=self.extensions,
-                output_format='html5',
+                output_format='xhtml',
```

Comment:
We should be careful with changes like this in such a big PR. If something like this causes a regression it will be hard to debug. 

(No need to change it back)